Frontend/Calculator.md

```diff
@@ -0,0 +1,34 @@
+# Interview Technical Assignment - Calculator Interface
+
+We ask all engineers to complete a technical test as part of our recruitment process at NICE. It's a great way for us to assess your technical abilities.
+
+We'll anonymously review your solution before inviting you for interview. Your test would then also form part of the interview itself: we'll discuss your approach, code and tests.
+
+We look for a number of things when we're reviewing, including:
+  - specific front-end coding skills
+  - understanding of things like progressive enhancement, performance and accessibility
+  - use of tooling and the npm ecosystem
+  - how you structure an application.
```

Comment:
```suggestion
  - how you structure an application
```

Reply:
@imranazad the full stop is deliberate! (see https://www.nice.org.uk/corporate/ecd1/chapter/punctuation-and-bullet-points#bullet-points)

Reply:
@ediblecode Ah wow, you learn something new everyday! Is that for accessibility reasons?